tasks/main.yml

```diff
@@ -3,35 +3,120 @@
 
 - include_vars: "{{ ansible_os_family }}.yml"
 
+- assert:
+    msg: dpb_cvsroot must not be empty
+    that:
+      - dpb_cvsroot is defined
+      - dpb_cvsroot | length > 0
+
+- name: Get release version
+  command: uname -r
+  register: register_release_version
+  changed_when: false
+
+- set_fact:
+    dpb_release: "{{ register_release_version.stdout }}" # x.y
```

Comment:
do you mean we should always leave URLs to official documentation? the example is mentioned in the OpenBSD FAQ, and I assume everyone has read it. the point that should be emphasized here is not the command, but why the fact is not used. I often leave URLs when it's hard to notice or realize meanings of something, more than anyone else, but `uname` and `cut`? facts change. the commands won't. do you really would like to know where the commands have been taken from?